ofrak_core/ofrak/core/patch_maker/modifiers.py

```diff
@@ -115,7 +121,19 @@ async def modify(self, resource: Resource, config: PatchFromSourceModifierConfig
         target_linkable_bom_info = await target_program.make_linkable_bom(
             patch_maker,
             build_tmp_dir,
+            patch_bom.unresolved_symbols,
         )
+
+        try:
```

Comment:
this is duplicate code from LinkableBinaryAnalyzer, can we just re-run the analyzer?